app/controllers/api/v2/hostgroups_controller.rb

```diff
@@ -40,6 +40,8 @@ def show
           param :subnet_id, :number, :desc => N_('Subnet ID')
           param :domain_id, :number, :desc => N_('Domain ID')
           param :realm_id, :number, :desc => N_('Realm ID')
+          param :config_group_ids, Array, :desc => N_("IDs of associated config groups")
+          param :config_group_names, Array, :desc => N_("Names of associated config groups")
```

Comment:
should the [same thing](https://github.com/theforeman/foreman/pull/4176#discussion_r95616147) be applied here and in app/controllers/api/v2/hosts_controller.rb ?